life_server/Functions/Gangs/fn_updateGang.sqf

```diff
@@ -5,44 +5,43 @@
     Description:
     Updates the gang information?
 */
-private ["_groupID","_bank","_maxMembers","_members","_membersFinal","_query","_owner"];
 params [
     ["_mode",0,[0]],
     ["_group",grpNull,[grpNull]]
 ];
 
 if (isNull _group) exitWith {}; //FAIL
 
-_groupID = _group getVariable ["gang_id",-1];
+private _groupID = _group getVariable ["gang_id",-1];
 if (_groupID isEqualTo -1) exitWith {};
 
 switch (_mode) do {
     case 0: {
-        _bank = [(_group getVariable ["gang_bank",0])] call DB_fnc_numberSafe;
-        _maxMembers = _group getVariable ["gang_maxMembers",8];
-        _members = [(_group getVariable "gang_members")] call DB_fnc_mresArray;
-        _owner = _group getVariable ["gang_owner",""];
+        private _bank = [(_group getVariable ["gang_bank",0])] call DB_fnc_numberSafe;
+        private _maxMembers = _group getVariable ["gang_maxMembers",8];
+        private _members = [(_group getVariable "gang_members")] call DB_fnc_mresArray;
+        private _owner = _group getVariable ["gang_owner",""];
         if (_owner isEqualTo "") exitWith {};
 
-        _query = format ["UPDATE gangs SET bank='%1', maxmembers='%2', owner='%3' WHERE id='%4'",_bank,_maxMembers,_owner,_groupID];
+        private _query = format ["UPDATE gangs SET bank='%1', maxmembers='%2', owner='%3' WHERE id='%4'",_bank,_maxMembers,_owner,_groupID];
```

Comment:
This file is currently relying on _query being defined outside the switch scope (albeit nil). See the last three lines of the file. _query needs to be declared (but not defined) in the parent scope